rbenv/riscv64.patch

```diff
@@ -0,0 +1,22 @@
+diff --git PKGBUILD PKGBUILD
+index f16f5ce..588e843 100644
+--- PKGBUILD
++++ PKGBUILD
+@@ -12,7 +12,7 @@ pkgdesc="Manage your app's Ruby environment"
+ arch=('any')
+ url="https://github.com/rbenv/rbenv"
+ license=('MIT')
+-depends=('bash')
++depends=('bash' 'git')
```

Comment:
I guess `git` should be in checkdepends from the [failing log](https://archriscv.felixc.at/.status/log.htm?url=logs/rbenv/rbenv-1.3.2-1.log).